bin/crew

```diff
@@ -1924,8 +1924,7 @@ end
 
 load_json
 
-@last_update_check = `stat -c %Y #{File.join(CREW_LIB_PATH, '.git/FETCH_HEAD')}`.chomp.to_i
-@last_update_check = @last_update_check.blank? ? 0 : @last_update_check
+@last_update_check = File.file?(File.join(CREW_LIB_PATH, '.git/FETCH_HEAD')) ? `stat -c %Y #{File.join(CREW_LIB_PATH, '.git/FETCH_HEAD')}`.chomp.to_i : `stat -c %Y #{File.join(CREW_LIB_PATH, 'lib/const.rb')}`.chomp.to_i
```

Comment:
We can use `File.mtime` here which returns a Time object, saving us some hassle and allowing us to compare things more easily. (mostly just putting this here to remind myself)

Reply:
Just did that in https://github.com/chromebrew/chromebrew/pull/10309/commits/529e15010863edd0e1ce81ee52cdd8b5d4b955b6